src/EditorFeatures/Core/Microsoft.CodeAnalysis.EditorFeatures.csproj

```diff
@@ -40,7 +40,12 @@
     <PackageReference Include="Microsoft.VisualStudio.ImageCatalog" Version="$(MicrosoftVisualStudioImageCatalogVersion)" PrivateAssets="all" />
     <PackageReference Include="Microsoft.VisualStudio.Language" Version="$(MicrosoftVisualStudioLanguageVersion)" />
     <!-- No warn on NU1701 until the LSP client targets netstandard2.0 https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1369985/ -->
-    <PackageReference Include="Microsoft.VisualStudio.LanguageServer.Client" Version="$(MicrosoftVisualStudioLanguageServerClientVersion)" NoWarn="NU1701" />
+    <PackageReference Include="Microsoft.VisualStudio.LanguageServer.Client" Version="$(MicrosoftVisualStudioLanguageServerClientVersion)" PrivateAssets="all" NoWarn="NU1701" />
```

Comment:
@sharwell I think there is a version of the LSP client targeting .netstandard now (17.2.39)
I will checkout your branch and test it to see if it removes the warnings.

Reply:
➡️ The package may be available, but we can't adopt it prior to 17.2 public release.

Reply:
FYI: All these will go away with my PR https://github.com/dotnet/roslyn/pull/59316

Reply:
Actually, i'm not changing EditorFeatures. I thought this is another project.